portworx-install-kubernetes/step1.md

```diff
@@ -16,7 +16,7 @@ Hint:
 # Check the k8s version
 >>Q2: What is the version of Kubernetes installed in this environment? << 
 () 1.13.0 
-(*) 1.11.3
+(*) 1.14
```

Comment:
How do we prevent this in the future? Can we get updates when changes are made? How do we found out who updated the backend k8as version?